AndelaOSP/Andela-Socials
# #165058763 Add dynamic fixtures

#### What Does This PR Do?

- Adds a simple infrastructure for creating dynamic fixtures in the app (in development environments). Also implements a dynamic fixture for the Event resource. This will improve the developer experience when devs need to seed data whose visibility is time or resource dependent.

#### Description Of Task To Be Completed

- [x] Install dynamic fixtures app for Django
- [x] Setup dynamic fixture for event

#### Any Background Context You Want To Provide?
*Current Behaviour on Develop Branch*

All fixtures are currently static

#### How can this be manually tested?
- Setup the project with Docker as outlined in the README
- Run `make build`
- Start the docker server container using `docker start dev_andela_social_server`
- Activate shell in the container using `docker exec -it dev_andela_social_server bash`
- In bash, run `python manage.py load_dynamic_fixtures`
- You should get a success message and can also confirm in your database that the fixture has been loaded

#### What are the relevant pivotal tracker stories?
[#165058763](https://www.pivotaltracker.com/story/show/165058763)

#### Screenshot(s)
 - N/A
